apps/arena/lib/arena/game_updater.ex

```diff
@@ -653,8 +660,8 @@ defmodule Arena.GameUpdater do
       |> Map.put(:external_wall, Entities.new_external_wall(0, config.map.radius))
       |> Map.put(:zone, %{
         radius: config.map.radius,
-        enabled: false,
-        shrinking: false,
+        enabled: config.game.zone_enabled,
+        shrinking: config.game.zone_enabled,
```

Comment:
I think the shriking should always be false, this isn't related to the zone being enabled or not, it's related to the zone shrinking

Reply:
You're right done.